modules/installation-osp-kuryr-octavia-upgrade.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-kuryr-octavia-configure_{context}"]
+= Using the Octavia OVN load balancer provider driver with Kuryr SDN
+
+If your {product-title} cluster uses Kuryr and was installed on a {rh-openstack-first} 13 cloud
+that was later upgraded to {rh-openstack} 16, you can configure it to use the Octavia OVN provider driver.
+
+[CAUTION]
+====
+Kuryr replaces existing load balancers after you change provider drivers. This process
+results in some downtime.
+====
+
+.Prerequisites
+
+* Install the {rh-openstack} CLI, `openstack`.
+
+* Install the {product-title} CLI, `oc`.
+
+* Verify that the Octavia OVN driver on {rh-openstack} is enabled.
++
+[TIP]
+====
+To view a list of available Octavia drivers, on a command line, enter link:https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.0/html/command_line_interface_reference/loadbalancer#loadbalancer_provider_list[`openstack loadbalancer provider list`].
+
+You should see the `ovn` driver in the command's output.
```

Comment:
If accurate, I'd be more direct and say something like, "The `ovn` driver is displayed in the command output."